src/components/Doctors/MapEvents.js

```diff
@@ -16,6 +19,23 @@ const MapEvents = function MapEvents() {
         searchValue,
       });
       setDoctors(mapDoctors);
+      const newZoom = map.getZoom();
+      const newCenter = map.getCenter();
+
+      const oldHash = location.hash.replace('#', '').split('|');
+
+      const oldHashValues = oldHash?.reduce(reduceHash, {
+        search: null,
+        loc: null,
+        zoom: null,
+        accepts: null,
+      });
+
+      const hash = `a-${oldHashValues.accepts}|l-${newZoom}/${[newCenter.lat, newCenter.lng].join(
```

Comment:
Coordinates are unnecessarily long, they should be rouded to 5 decimals (1m precision)

Reply:
Tnx for https://github.com/sledilnik/zdravniki/pull/211/commits/6e38949ff1c89643d75237be33081b61d2ff7f87